test/components/Col.spec.js

```diff
@@ -17,6 +17,11 @@ describe('Col', () => {
     expect(className).toContain(style['col-lg-4']);
   });
 
+  it('Should add "first-*" class if "first" property is set', () => {
+    renderer.render(<Col first='md'/>);
```

Comment:
Use double quotes for prop values (`<Col first="md" />`), I think this is the only reason why tests fail.